src/content/configuration/externals.md

```diff
@@ -10,9 +10,9 @@ contributors:
   - zefman
 ---
 
-The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's environment. This feature is typically most useful to __library developers__, however there are a variety of applications for it.
+The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's environment. This feature is typically most useful to __library developers__, however, there are a variety of applications for it.
```

Comment:
I'm not sure whether it would make more sense if we would start a new sentence at `However` part.

Reply:
I think, with a new line, it will make more sense. So, should I change it?

Reply:
I mean I guess it's fine even the way it is now. Not a big deal. Thanks!

Reply:
@byzyk reverted